src/interfaces/SpokePool.ts

```diff
@@ -19,6 +19,7 @@ export interface RelayData {
   fillDeadline: number;
   exclusiveRelayer: string;
   exclusivityDeadline: number;
+  _hash?: Record<number, string>;
```

Comment:
It'd be a fair bit more efficient for this to be a simple string, but the `RelayData` type doesn't include the `destinationChainId`, so the same RelayData object can map to multiple hashes depending on the value of `destinationChainId`. In the interests of safety it seems best to use a record, but I'd love to find a way to use a string instead.

Reply:
Does that mean the hash really should be one level higher where the destination chain is present (i.e. the hash should be in fill and deposit)?

Reply:
Yeah, that's a good option. It introduces a little bit of duplication but it's nothing major. I'll think about another way of passing in the hash if it's not defined in `RelayData`.